services/rook-ceph-cluster/1.10.8/objectbucketclaims/helmrelease.yaml

```diff
@@ -12,6 +12,7 @@ spec:
         name: mesosphere.github.io-charts-stable
         namespace: kommander-flux
       version: 0.1.4
+  timeout: 20m
```

Comment:
I bumped this because the this HR was timing out while running the health check jobs (waiting for the OBCs to be ready). We won't really need this once we move away from the obc health check jobs